deploy/eks-terraform/kiam.tf

```diff
@@ -0,0 +1,51 @@
+#
+# Create the intermediary Kiam role
+#
+resource "aws_iam_role" "kiam_intermediary" {
+  name        = "eks-${var.cluster_name}-kiam-intermediary"
+  description = "Role the Kiam server process assumes. If you want to use Kiam to assume any role R, THIS is the role that R must trust."
```

Comment:
I would maybe rephrase this slightly, something like "Role the Kiam server process assumes. Any role that you want Kiam to assume must trust **THIS** role."